profiles/schema.py

```diff
@@ -366,6 +373,39 @@ def mutate(self, info, **kwargs):
         return UpdateProfile(profile=profile)
 
 
+class DeleteProfile(graphene.Mutation):
+    ok = graphene.Boolean()
+
+    class Arguments:
+        id = graphene.ID(required=True)
+
+    def mutate(self, info, **kwargs):
```

Comment:
you could probably ensure that there is a user in the context with `@login_required`

Reply:
Thanks for spotting. I lost it somewhere 😸